pkg/kube/inject/testdata/inject/app_probe/https-probes.yaml.injected

```diff
@@ -14,10 +14,7 @@ spec:
   template:
     metadata:
       annotations:
-        sidecar.istio.io/interceptionMode: REDIRECT
```

Comment:
I know it's not related to this PR, but not injecting the `sidecar.istio.io/interceptionMode` annotation anymore is breaking Istio CNI. It relies on it. If it's not present, it will always default to `REDIRECT` mode.
This is a regression in the new installer.